.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,173 @@
+        required: false
+      load:
+        description: 'Specifies which benchmark components to deploy. `starter`, `timer` and `publisher` can be assigned with the rate at which they publish'
+        default: 'worker, starter=200'
+        required: false
+  workflow_call:
+    inputs:
+      name:
+        description: 'Specifies the name of the benchmark'
+        type: string
+        required: true
+      ref:
+        description: 'Specifies the ref (e.g. main or a commit sha) to benchmark'
+        default: 'main'
+        type: string
+        required: false
+      cluster:
+        description: 'Specifies which cluster to deploy the benchmark on'
+        default: 'zeebe-cluster'
+        type: string
+        required: false
+      cluster-region:
+        description: 'Specifies the cluster region. Needed to retrieve cluster credentials'
+        default: europe-west1-b
+        type: string
+        required: false
+      load:
+        description: 'Specifies which benchmark components to deploy. `starter`, `timer` and `publisher` can be assigned with the rate at which they publish'
+        default: 'worker, starter=200'
+        type: string
+        required: false
+jobs:
+  build-zeebe-image:
+    name: Build zeebe image
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - run: gcloud auth configure-docker
+      - uses: ./.github/actions/setup-zeebe
+        with:
+          maven-cache: 'true'
+      - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
+        with:
+          maven-extra-args: -T1C
+      - uses: ./.github/actions/build-docker
+        with:
+          repository: gcr.io/zeebe-io/zeebe
+          version: ${{ inputs.name }}
+          push: true
+          distball: ${{ steps.build-zeebe.outputs.distball }}
+  deploy-benchmark-cluster:
+    name: Deploy benchmark cluster
+    needs:
+      - build-zeebe-image
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - uses: google-github-actions/get-gke-credentials@v0.8.2
+        with:
+          cluster_name: ${{ inputs.cluster }}
+          location: ${{ inputs.cluster-region }}
+      - uses: azure/setup-helm@v3
+      - name: Add camunda helm repo
```

Comment:
:wrench: Personally I would prefer to have script which contains these parts, so we can also run that locally. Because I guess you want to remove the other scripts?

Reply:
Does anything speaks against using our already existing makefiles, scripts?